crates/data/src/lib.rs

```diff
@@ -117,3 +117,75 @@ pub struct Summary<'a> {
     /// The mean deviation (note: not standard deviation) of the `count` field.
     pub mean_deviation: f64,
 }
+
+/// The effect size (and confidence interval) of between two different engines
```

Comment:
```suggestion
/// The effect size (and confidence interval) between two different engines
```
Maybe this is what is meant here?